component/http/cache/cache_metrics.go

```diff
@@ -0,0 +1,69 @@
+package cache
```

Comment:
since we are in the cache package could we can name the file just metric.go (we should be using the singular name convention)